src/olympia/hero/models.py

```diff
@@ -88,7 +90,32 @@ def get_choices(self, *args, **kwargs):
         return super().get_choices(*args, **kwargs)
 
 
+class PrimaryHeroImage(models.Model):
+    custom_image = models.ImageField(
+        upload_to='static/img/hero/featured',
+        blank=True)
+
+    def __str__(self):
+        return f'{self.custom_image}'
+
+    def preview_image(self):
+        return mark_safe('<img src="{}" width="150" />'.format(
+            self.custom_image.url)
+        )
+    preview_image.short_description = "Image"
+    preview_image.allow_tags = True
+
+
+class ImageChoiceField(forms.ModelChoiceField):
+    def label_from_instance(self, obj):
+        return mark_safe(
+            '<img src="{}" width="150" height="120" />'.format(
+                obj.custom_image.url))
+
+
 class PrimaryHero(ModelBase):
+    select_image = models.ForeignKey(
+        PrimaryHeroImage, blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
`blank` only really makes sense for text type fields -  it's not used for foreign keys

Reply:
I had forgotten to take out `blank` when I was trying to figure out how to not make `selected_image` a required field. Makes sense why that didn't work now, thanks for the clarification! It has now been removed. :)